mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -28,6 +28,15 @@ protected-branches-extra-contexts: "cla/linuxfoundation"
 
 # submit-queue options. Keep job lists sorted!
 protected-branches: "master"
+presubmit-jenkins-jobs: "\
```

Comment:
I'm not sure, I thought of that but I didn't want to incorrectly couple it. I think now that it makes sense to just use `require-retest-contexts`.